src/lib/PageControls.js

```diff
@@ -220,7 +280,7 @@ class PageControls extends EventEmitter {
 
             case 'Escape':
                 this.hidePageNumInput();
```

Comment:
I'll verify this on Monday but I believe there's an IE 11 issue when hitting escape in fullscreen that also closes the preview. Pretty sure this is coming from the webapp side as they own that shortcut.